data/mods/ssb/items.ts

```diff
@@ -1,4 +1,15 @@
 export const BattleItems: {[k: string]: ModdedItemData} = {
+	// phiwings99
+	boatiumz: {
+		name: "Boatium Z",
+		isNonstandard: "Custom",
+		onTakeItem: false,
+		zMove: "Ghost of 1v1 Past",
+		zMoveFrom: "Destiny Bond",
+		itemUser: ["Froslass"],
+		gen: 8,
+		desc: "If held by a Froslass with Drain Punch, it can use Ghost of 1v1 Past.",
```

Comment:
Drain Punch?

Reply:
i literally typed this manually, im not sure how i did that